assets/css/edd-admin.css

```diff
@@ -2431,6 +2414,57 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div.edd-reports-export {
+	display: grid;
+	display: -ms-grid;
+	grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+	grid-gap: 20px;
+}
+
+.edd-reports-wrapper .postbox {
+	margin-bottom: 0;
+}
+
+.edd-export-sales-earnings {
+	grid-template-columns: 1fr;
+}
+
+[class^="edd-export"] .edd-from-to-wrapper {
```

Comment:
Should this use the `.edd-reports-export` class instead of `[class^="edd-export"]`?